src/popup/locales/en.json

```diff
@@ -337,6 +337,7 @@
       "recipientPlaceholder": "Enter public address or .chain name",
       "recipientPlaceholderUrl": "Enter public address, .chain name or URL",
       "recipientPlaceholderProtocol": "Enter {name} public address",
+      "recipientPlaceholderEth": "Enter Ethereum public address or ENS name",
```

Comment:
I strongly suggest to not create any protocol related translations. In my opinion we need to be open for new protocols and make the translations in a generic way.

Reply:
With the current design reference we have to create atleast one protocol specific translation for "or ENS name"